libsolidity/codegen/ContractCompiler.cpp

```diff
@@ -520,7 +520,8 @@ bool ContractCompiler::visit(FunctionDefinition const& _function)
 bool ContractCompiler::visit(InlineAssembly const& _inlineAssembly)
 {
 	ErrorList errors;
-	assembly::CodeGenerator codeGen(errors);
+	ErrorReporter errorReporter(errors);
```

Comment:
It was added recently with assembly refactorings. it needs the `ExternalIdentifierAccess::Resolver`, probably a forward declaration for that will be sufficient, otherwise including `AbstractAssembly.h` would be needed.